go/tools/gopackagesdriver/bazel_json_builder.go

```diff
@@ -49,6 +51,29 @@ func (b *BazelJSONBuilder) fileQuery(filename string) string {
 		label = fmt.Sprintf("@%s//%s", matches[1], strings.Join(matches[2:], ":"))
 	}
 
+	relToBin, err := filepath.Rel(b.bazel.info["bazel-bin"], filename)
+	if err == nil && !strings.Contains(relToBin, "../") {
+		fmt.Fprintln(os.Stderr, "File is under bazel-bin, reverse walking tree to find matching BUILD.bazel.")
```

Comment:
Who would see this message? Could it be noisy?

Reply:
Fair, I figured since it might not be the expected flow I'd highlight it, but I guess we don't do that anywhere.